md/bonita-docker-installation.md

```diff
@@ -257,4 +244,20 @@ The procedure to migrate a Bonita container is therefore as follow:
 * Stop and destroy the running Bonita container.
 * Play the migration script on your Bonita database see [migrate the platform from an earlier version of Bonita](migrate-from-an-earlier-version-of-bonita-bpm.md#migrate).
 * Get the new Bonita docker image, as explained above.
+* Update the license, see [Update configuration and license section](#section-update-configuration)
 * Start a new Bonita container.
+
+
+<a id="section-update-configuration" />
+
+## Update configuration and license
+
+The license file and configuration files are updated using the Setup tool.
+
+Setup tool can be used outside of the Docker container directly by downloading the Tomcat bundle and running it from there.
+
+::: info
+The setup tool need to be able to access the database. Because of that, if the database is in a docker container, its port must be exposed to the host.
```

Comment:
```suggestion
The setup tool needs to be able to access the database. Because of that, if the database is in a docker container, its port must be exposed to the host.
```